charts/keycloak-bootstrap/templates/bootstrap-configmap.yaml

```
@@ -17,4 +18,4 @@ data:
 {{ toYaml .Values.keycloak.preloadedClients | indent 4 }}
   users.yaml: |-
 {{ toYaml .Values.keycloak.preloadedUsers | indent 4 }}
-
+{{- end }}
```

Fail if the any of the above values are set with an explanation 
```suggestion
{{- else if or attributes.preloadedAttributes attributes.preloadedAuthorities entitlements.realms keycloak.customConfig keycloak.preloadedClients keycloak.preloadedUsers }}
{{- fail "Conflicting values: setting `existingConfigSecret` ignores other configuration input" }}
{{- end }}
```